components/gitlab/actions/create-branch/create-branch.mjs

```diff
@@ -1,38 +1,39 @@
-// legacy_hash_id: a_q1iol0
-import { axios } from "@pipedream/platform";
+import gitlab from "../../gitlab.app.mjs";
 
 export default {
   key: "gitlab-create-branch",
   name: "Create Branch",
-  description: "Create a new branch in the repository.",
-  version: "0.2.1",
+  description: "Create a new branch in the repository. [See docs](https://docs.gitlab.com/ee/api/branches.html#create-repository-branch)",
+  version: "0.3.0",
```

Comment:
Since the previous version was "0.2.1", I think the version should be "0.2.2".
```suggestion
  version: "0.2.2",
```

Reply:
Since there are structural changes in how the actions and app work, and are not just bug fixes, I think [changing the minor version would be ideal](https://pipedream.com/docs/components/guidelines/#versioning).